Marlin/Conditionals_LCD.h

```diff
@@ -360,8 +360,12 @@
     #if ENABLED(Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN)
       #undef Z_MIN_ENDSTOP_INVERTING
       #define Z_MIN_ENDSTOP_INVERTING false
+      #undef Z_MIN_PROBE_ENDSTOP_INVERTING
+      #define Z_MIN_PROBE_ENDSTOP_INVERTING false
       #define TEST_BLTOUCH() _TEST_BLTOUCH(Z_MIN)
     #else
+      #undef Z_MIN_PROBE_ENDSTOP_INVERTING
+      #define Z_MIN_PROBE_ENDSTOP_INVERTING false
```

Comment:
> Should this be the case for all BLTouch, regardless of whether Z_MIN_ENDSTOP_INVERTING is being forced to false?

Yes. BLTouch requires that the input be non-inverting. If `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN` is enabled then `Z_MIN_ENDSTOP_INVERTING` and `Z_MIN_PROBE_ENDSTOP_INVERTING` are defined as `false` to appease the sanity check. If `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN` is not enabled, then `Z_MIN_ENDSTOP_INVERTING` should be left alone since the printer is using an endstop that may or may not need to be inverted, but `Z_MIN_PROBE_ENDSTOP_INVERTING` should still be defined as `false` for the BLTouch.